docs/features.rst

```diff
@@ -3,6 +3,10 @@ Features
 
 ``pip-compile-multi`` supports many options to customize compilation.
 Each option can be specified in ``requirements.ini`` file, by replacing dashes with underscores.
+Options can also be specified in the ``pyproject.toml`` file by going adding the ``[tool.pip-compile-multi.requirements]``
+section and defining the same options there using TOML syntax. If the TOML file is available with that section defined,
+it will take precedence over other pip-compile-multi configuration files
```

Comment:
> If the TOML file is available with that section defined, it will take precedence over other pip-compile-multi configuration files

I wonder if this design is necessary. Can `pyproject.toml` be considered equal to the rest `'requirements.ini', 'setup.cfg', 'tox.ini'` and have sections from it appended to the final result, instead of replacing it?